dissect/target/plugins/os/windows/catroot.py

```diff
@@ -217,12 +217,11 @@ def catdb(self) -> Iterator[CatrootRecord]:
             with ese_file.open("rb") as fh:
                 ese_db = EseDB(fh)
 
-                tables = [table.name for table in ese_db.tables()]
                 for hash_type, table_name in [("sha256", "HashCatNameTableSHA256"), ("sha1", "HashCatNameTableSHA1")]:
-                    if table_name not in tables:
+                    if not (table := ese_db.table(table_name)):
```

Comment:
This is actually an EseDB which unfortunately raises a KeyError if a table does not exist.